packages/design-system-docs/src/pages/components/Accordion/_Accordion.docs.scss

```diff
@@ -0,0 +1,73 @@
+/*
+Accordion
+
+@uswds https://designsystem.digital.gov/components/accordion/
+
+An accordion is a list of headers that hide or reveal additional content when selected.
+
+Markup: accordion.example.html
+
+@status Draft
+
+Style guide: components.accordion
+*/
+
+/*
+`<Accordion>`
+
+@react-example Accordion.example.jsx
+
+@react-props Accordion.tsx
+
+Style guide: components.accordion.react
+*/
+
+/*
+`<AccordionItem>`
+
+`AccordionItem` renders the Accordion element.
+
+@react-props AccordionItem.tsx
+
+Style guide: components.accordion.react-accordionitem
+*/
+
+/*
+---
+
+### Style customization
+
+- **$accordion-border-color** -  Border color of the accordion
+- **$accordion-header-bg-color** - Background color of the accordion header
+- **$accordion-header-bg-hover-color** - Background color of the accordion header when hovered
+
+### Keyboard support
+- **header** - *Enter or Spacebar* - Changes the collapsed/expanded state of the associated accordion panel
+- **header** - *Arrow Down* - Moves focus to the next accordion header
+- **header** - *Arrow Up* - Moves focus to the previous accordion header
+
+### When to use
+
+- Use an accordion to see an overview of multiple, related sections of content
+- Use an accordion to show and hide sections as needed
+
+### When to consider alternatives
+
+Accordions hide content from users and not everyone will notice them or understand how they work.
+Accordions increase cognitive load and interaction cost, as users have to make decisions about what headers to click on.
+
+- Don't use an accordion if users need to see most or all of the information on a page.
+- Don't use an accordion if there is not enough content to warrant condensing.
+
+### Usage
```

Comment:
Based on the conversation here in the comments, do we need to state here that we do not recommend nesting accordions?

Reply:
Working with CMS folks to finalize accordion documentation here - https://confluence.cms.gov/display/HCDSG/Accordion+guidance